pyproject.toml

```diff
@@ -81,6 +81,8 @@ docs = [
   "sphinx-copybutton<=0.5.2",
   "sphinxcontrib-bibtex<=2.64",
   "jupyter-sphinx<=0.5.3",
+  "qiskit>=2.1.1",
+  "topologiq@git+https://github.com/tqec/topologiq.git"
```

Comment:
since you added the integration group, you can directly use it here https://packaging.python.org/en/latest/specifications/dependency-groups/

Reply:
See the example at the provided link:

<img width="833" height="414" alt="Image" src="https://github.com/user-attachments/assets/28fd710f-7b09-4532-94c1-a46398573dd3" />